src/calendar/calendar.api.json

```diff
@@ -165,7 +175,7 @@
     {
       "name": "disabledDaysIndexes",
       "type": "number[]",
-      "description": "Whether to apply custom disable color to selected day names indexes"
+      "description": "Whether to apply custom disable color to selected day names indexes (Sunday=0)"
```

Comment:
if `firstDay` is not 0 then the Sunday is not 0.

Reply:
That's a good point but not correct. Date().getDay() returns the day of the week index (Sun = 0) and the date-state-manager is checking against that in the disabledByWeekDays feature. So no matter what the first day is the date is always on the same day of the week (ex. 20-11-2024 is always Wednesday).

Reply:
Just making sure, what do we expect to happen here?
```
firstDay={1}
disabledDaysIndexes={[0]}
```
This is the result. Monday, which is in index 0, is shown as disabled.
![image](https://github.com/user-attachments/assets/91bf0189-967c-42cd-8d7f-37ecf74c8d88)

Reply:
If you remove the markingType prop, I'll expect that the first day displayed will be Monday and alll the Sundays will be disabled.
You're checking on a period marking... as mentioned, there's an issue since basic day and period day are not the same.

Reply:
Sunday have to be 0. This is a bug that need to be fixed, but fixing it is a breaking change so I'll add it to the next major version